api/ceramic_cache/api/v1.py

```diff
@@ -498,7 +498,7 @@ def handle_get_ui_score(
             score.expiration_date is not None and score.expiration_date < now
         ):
             # This will re-calculate the score and update the expiration date
-            ret = get_detailed_score_response_for_address(address)
```

Comment:
This is not actually part of the fix, correct?
It also does not really change anything.